src/staff-administrator/components/staff-add-edit-user/staff-add-edit-user-form/staff-add-edit-user-form.component.ts

```diff
@@ -46,7 +49,9 @@ export class StaffAddEditUserFormComponent implements OnInit, OnDestroy {
         if (data.reset) {
           this.resetForm();
         } else {
-          this.router.navigateByUrl('/staff/add-user/check-your-answers');
+          if (this.previousUrl !== '/staff/add-user/check-your-answers') {
+            this.router.navigateByUrl('/staff/add-user/check-your-answers');
```

Comment:
Can we have `/staff/add-user/check-your-answers` as `const `to reuse it?
We are using it in more than one place.